indico/modules/events/management/blueprint.py

```diff
@@ -45,6 +45,8 @@
 # Cloning
 _bp.add_url_rule('/clone', 'clone', cloning.RHCloneEvent, methods=('GET', 'POST'))
 _bp.add_url_rule('/clone/preview', 'clone_preview', cloning.RHClonePreview, methods=('GET', 'POST'))
+_bp.add_url_rule('/import', 'import', cloning.RHImportFromEvent, methods=('GET', 'POST'))
+_bp.add_url_rule('/import/event-details', 'import_event_details', cloning.RHImportEventDetails, methods=('GET', 'POST'))
```

Comment:
I don't think this one uses anything but POST

```suggestion
_bp.add_url_rule('/import/event-details', 'import_event_details', cloning.RHImportEventDetails, methods=('POST',))
```